hack/defaults

```diff
@@ -6,7 +6,7 @@ WAIT_TIMEOUT="${WAIT_TIMEOUT:-300s}"
 # Use upstream manifests with downstream containers
 KUBEVIRT_MANIFEST_VERSION="v0.17.0"
 CDI_MANIFEST_VERSION="v1.6.0"
-NETWORK_ADDONS_MANIFEST_VERSION="0.7.0"
+NETWORK_ADDONS_MANIFEST_VERSION="0.9.0"
```

Comment:
should be need to add also the nmstate to this file as NETWORK_ADDONS_NMSTATE_STATE_HANDLER_IMAGE ?

Reply:
Oh, nice catch! You are right